src/_webp.c

```diff
@@ -15,10 +13,8 @@
  * very early versions had some significant differences, so we require later
  * versions, before enabling animation support.
  */
-#if WEBP_MUX_ABI_VERSION >= 0x0104 && WEBP_DEMUX_ABI_VERSION >= 0x0105
-#define HAVE_WEBPANIM
-#endif
-
+#if WEBP_MUX_ABI_VERSION < 0x0106 || WEBP_DEMUX_ABI_VERSION < 0x0107
+#error libwebp 0.5.0 and above is required. Upgrade libwebp or build with --disable-webp flag
```

Comment:
```suggestion
#error libwebp 0.5.0 and above is required. Upgrade libwebp or build Pillow with --disable-webp flag
```
Just clarifying that it's not libwebp that is being built with `--disable-webp`